apps/condo/domains/property/schema/Property.js

```diff
@@ -47,16 +55,23 @@ const Property = new GQLListSchema('Property', {
             hooks: {
                 validateInput: async ({ resolvedData, fieldPath, addFieldValidationError, context, existingItem, operation }) => {
                     const value = resolvedData[fieldPath]
-                    let propertiesWithSameAddressInOrganization
-                    if (operation === 'create') {
-                        propertiesWithSameAddressInOrganization = await PropertyAPI.getAll(context, { address: value, organization: { id: resolvedData.organization }, deletedAt: null })
-                    }
-                    else if (operation === 'update' && resolvedData.address !== existingItem.address) {
-                        propertiesWithSameAddressInOrganization = await PropertyAPI.getAll(context, { address: value, organization: { id: existingItem.organization }, deletedAt: null })
-                    }
+                    const isCreate = operation === 'create'
+                    const isUpdateAddress = operation === 'update' && resolvedData.address !== existingItem.address
+
+                    if (isCreate || isUpdateAddress) {
```

Comment:
if you want to refactor it, please also add tests for this two cases!

Reply:
Actually we already have tests for these cases.

![image](https://user-images.githubusercontent.com/91055869/145260696-3d2ed074-bb77-4afc-84db-87a821c2cf82.png)

![image](https://user-images.githubusercontent.com/91055869/145260763-8b7a0257-d233-4a79-af52-20dabc722bda.png)